docs/docs/bootstrapping.md

```diff
@@ -0,0 +1,61 @@
+<div class="content-title">Bootstrapping</div>
+
+***
```

Comment:
I can fix up documents to my tweak-level after merge, but I'd like a summary of what bootstrapping is/does.